stablehlo/dialect/StablehloOps.td

```diff
@@ -202,7 +202,7 @@ def StableHLO_AbsOp: StableHLO_UnaryElementwiseOp<"abs",
     [Pure,
      DeclareOpInterfaceMethods<InferTypeOpInterface>],
      TensorOf<[HLO_SInt, HLO_Float, HLO_Complex]>> {
-  let summary = "Absolute value operator";
+  let summary = "Absolute value operation";
```

Comment:
Abs is a shorthand, in case of `SqrtOp` we will write `Square root operation`

Reply:
WDYT

Reply:
I would suggest that we follow the `<Op name without "Op"> operation` format, so that we don't have to think what to write here, e.g. sin vs sine, tanh vs hyperbolic tangent, etc.

Reply:
For additional background on this, there is a history of slight mismatches between op names and op mnemonics, e.g. `MulOp` has the `stablehlo.multiply` mnemonic, and this is quite confusing. Given that, I think it would be desirable to pick a single name and stick to it everywhere - in the name, in the mnemonic, in the summary, etc.

We can't easily change the names and the mnemonics (there is a ticket for that - #351 - and there'll be a lot of work to address it), but I would recommend that we don't introduce new divergencies.

Reply:
thanks.